pom.xml

```diff
@@ -78,11 +78,11 @@
 
 	<modules>
 		<module>runtime/Java</module>
+        <module>runtime/Kotlin</module>
 		<module>tool</module>
 		<module>antlr5-maven-plugin</module>
 		<module>tool-testsuite</module>
 		<module>runtime-testsuite</module>
-        <module>runtime/Kotlin</module>
```

Comment:
I'm not sure that order of included modules can affect compilations. Moreover, the current order is incorrect because `runtime-testsuite` module actually depends on `runtime/Kotlin`.

Reply:
Maven's Reactor will take care of ordering modules based on dependencies, so the order doesn't matter.